### PR TITLE
chore: Update language to id-ID

### PR DESCRIPTION
**Pull request by gaweki**
few language string to translate.

just in basic search word -> "cars", "jam(in english: clock)"

### CODE REVIEW

**Comment by gaweki** [comment]
@GioSensation sir, please review..

File: locales/id_ID/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -4947,7 +4947,7 @@ msgstr ""
 
 msgctxt "welcome message"
 msgid "You now have the DuckDuckGo extension!"
-msgstr ""
+msgstr "Anda sekarang memiliki ekstensi DuckDuckGo"
```
Comment (GioSensation) [inline locales/id_ID/LC_MESSAGES/duckduckgo.po:4950]: Forgot the exclamation mark.

```suggestion
msgstr "Anda sekarang memiliki ekstensi DuckDuckGo!"
```

File: locales/id_ID/LC_MESSAGES/duckduckgo.po [hunk 2]
```diff
@@ -4979,7 +4979,7 @@ msgstr ""
 
 msgctxt "precise_user_location"
 msgid "You've chosen not to set your location."
-msgstr ""
+msgstr "Anda telah memilih untuk tidak mengatur lokasi anda"
```
Comment (GioSensation) [inline locales/id_ID/LC_MESSAGES/duckduckgo.po:4982]: Forgot the closing period.

```suggestion
msgstr "Anda telah memilih untuk tidak mengatur lokasi anda."
```

**Comment by GioSensation** [comment]
Thanks for your contribution, @gaweki! Much appreciated. Merging now. 🎉

**Comment by gaweki** [comment]
when i can see new translation in production, mr @GioSensation ?

**Comment by GioSensation** [comment]
@gaweki We usually release daily, so you should see the translations live tomorrow. 🚀

**Pull request merged**